helm/ph-ee-engine/templates/connector-channel.yml

```diff
@@ -35,20 +63,32 @@ spec:
             name: ph-ee-config
 
 ---
-apiVersion: extensions/v1beta1
+{{- if .Values.channel.ingress.enabled }}
+apiVersion: {{ .Values.ingress.apiversion }}
 kind: Ingress
 metadata:
   name: ph-ee-connector-channel
   annotations:
-    kubernetes.io/ingress.class: azure/application-gateway
+{{- if .Values.channel.ingress.annotations }}
+{{ toYaml .Values.channel.ingress.annotations | indent 4 }}
+{{- end }}
 spec:
   rules:
-    - host: "{{ .Values.ph_ee_connector_channel.hostname }}"
+    - host: "{{ .Values.channel.hostname }}"
       http:
         paths:
-        - backend:
-            serviceName: ph-ee-connector-channel
-            servicePort: 80
+        - path: /channel
+          pathType: Prefix
+          backend:
+            service:
+              name: ph-ee-connector-channel
```

Comment:
Not sure if this will work.  Backend.servicename got changed to backend.service.name in latest k8s. If this chart works in IBM environment, then consider this comment resolved.